test/e2e/run_test.go

```diff
@@ -51,6 +51,7 @@ var _ = Describe("Podman run", func() {
 
 	It("podman run selinux grep test", func() {
 		selinux := podmanTest.SystemExec("ls", []string{"/usr/sbin/selinuxenabled"})
+		selinux.WaitWithDefaultTimeout()
```

Comment:
We should be able to do this much easier and more sensible by just importing go-selinux and then just checking if selinux enabled.  No need to exec a program.

Reply:
OK, will update this.